app/models/user.rb

```diff
@@ -4,9 +4,13 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  validates :first_name, :last_name, :is_chef, :address, :date_of_birth, presence: true
+  validates :first_name, :last_name, :address, :date_of_birth, presence: true
 
   has_many :reviews
   has_many :bookmarks
   has_many :messages
+
+  validates :bio, presence: true
+  validates :specialties, presence: true
```

Comment:
Lines 13 and 14 don't exist in the database, also we don't want to make this mandatory or all users will have to include this at sign-up. We want these attributes to be added at the update profile

Reply:
additional attributes haves been removed, following schema